### DISCUSSION TITLE
no  executable code or content only documents flag

### DISCUSSION THREAD

**Discussion by ati-ozgur** [quarto-dev/quarto-cli#11050] [Feature Requests]
### Description

Since my previous issue below rejected citing security issues.
https://github.com/quarto-dev/quarto-cli/issues/11039

I believe that a flag should be added for content only documents.
My documents did not contain any executable code apart from some mermaid documents.
But a flag in the quarto will resolve this issue about security beautifully.

**Comment by mcanouil** [comment]
> Since my previous issue below rejected citing security issues.

Nothing was rejected.
It was only ask of you to make your example small as we cannot spend time to investigate your 50 files project for you.
Such statements not only misrepresent the facts but also have the potential to harm.
We urge you to reconsider the information you have received and to cease spreading these false claims.
(cc @cscheid)

You mentioned spending several hours and you apparently were unable to reproduce with something small, why would you expect us to spend even more time to investigate a complex project we don't know anything about.
Note that multiplying issues/discussions does help us help you.

The usual practice of removing pieces one at a time is the way to debug anything including Quarto projects/documents.

Note that there is a 95% your issue is about your relative file paths and possibly the use of include/embed shortcode somewhere.

**Comment by mcanouil** [comment]
Also, I'm going to close this because, there is a flag for that already `quarto render --help`.
Even if there wasn't, it does not change the underlying issue and our request.

**Comment by ati-ozgur** [comment]
I only see following two parameters.
This does not mention that no code will be run.
May be updating the help so that it will be more understandable.

│  --execute                           - Execute code (--no-execute to skip execution).                                                 │
│  -P, --execute-param                 - Execution parameter (KEY:VALUE).

**Comment by ati-ozgur** [comment]
Also I am not multiplying anything.
These are my issues added after first discussion.
Everyone of them are about different topics:

1. This one about the security issue. You say it exists but help does not mention it explicitly.
2. [Please add some debugging examples after cloning git repo](https://github.com/quarto-dev/quarto-cli/issues/11043)[documentation](https://github.com/quarto-dev/quarto-cli/issues?q=author%3A%40me%20label%3Adocumentation) Your readme is only about installation. There is no help file for new developers to debug the code. 
3. [Please add some debugging examples after cloning git repo](https://github.com/quarto-dev/quarto-cli/issues/11043)[documentation](https://github.com/quarto-dev/quarto-cli/issues?q=author%3A%40me%20label%3Adocumentation) This is another issue about references. Not related to images.
4. First one is about images. 

I fail to see how I multiply issues.
I use quarto and I value quarto; thus, I am spending my time to enter issues.
